doc/TUTORIALS_CN.md

```diff
@@ -869,6 +871,266 @@ Rosetta 完整代码参考 [rtt-ds-lr.py](../example/tutorials/code/rtt-ds-lr.py
+            epoch_loss = 0.0
+            for batch in range(n_batches):
+                X_batch, Y_batch = tfs.run(train_next_iterator)
+                feed_dict = {x: X_batch, y: Y_batch}
+                _, batch_loss = tfs.run([optimizer, loss], feed_dict)
+                epoch_loss += batch_loss
+        
+            average_loss = epoch_loss / n_batches
+            print("epoch: {0:04d} loss = {1:0.6f}".format(
+                epoch, average_loss))
+        feed_dict = {x: X_test, y: Y_test}
+        accuracy_score = tfs.run(accuracy_function, feed_dict=feed_dict)
+        print("accuracy={0:.8f}".format(accuracy_score))
+        
+# 构造输入
+x = tf.placeholder(dtype=tf.float64, name="x", 
+                    shape=[None, num_inputs])
+# 构造输出
+y = tf.placeholder(dtype=tf.float64, name="y", 
+                    shape=[None, 10])
+#目前设置隐藏层为0，可自己修改
+num_layers = 2
+num_neurons = [128, 256]
+learning_rate = 0.01
+n_epochs = 30
+batch_size = 100
+n_batches = int(mnist.train.num_examples/batch_size)
+
+model = mlp(x=x,
+            num_inputs=num_inputs,
+            num_outputs=num_outputs,
+            num_layers=num_layers,
+            num_neurons=num_neurons)
+
+loss = tf.reduce_mean(
+    tf.nn.sigmoid_cross_entropy_with_logits(logits=model, labels=y))
+optimizer = tf.train.GradientDescentOptimizer(
+    learning_rate=learning_rate).minimize(loss)
+
+predictions_check = tf.equal(tf.argmax(model, 1), tf.argmax(y, 1))
+accuracy_function = tf.reduce_mean(tf.cast(predictions_check, dtype=tf.float64))
+#训练
+tensorflow_classification(n_epochs=n_epochs, 
+   n_batches=n_batches, 
+   batch_size=batch_size, 
+   model = model, 
+   optimizer = optimizer, 
+   loss = loss, 
+   accuracy_function = accuracy_function, 
+   X_test = X_test, 
+   Y_test = Y_test
+   )
+```
+
+完整代码参考[tf-mlp_mnist.py](../example/tutorials/code/tf-mlp_mnist.py)
+
+执行以下命令即可运行
+
+```python
+python ./tf-mlp_mnist.py
+```
+
+输出如下：
+
+```python
+epoch: 0000 loss = 17.504000
+epoch: 0001 loss = 6.774922
+epoch: 0002 loss = 4.993065
+epoch: 0003 loss = 4.047511
+epoch: 0004 loss = 3.440471
+epoch: 0005 loss = 3.006049
+  ...
+epoch: 0027 loss = 0.874316
+epoch: 0028 loss = 0.847307
+epoch: 0029 loss = 0.821776
+accuracy=0.91100000
+```
+
+#### Rosetta版
+
+- 导包并激活协议
+
+```python
+import os
+import tensorflow as tf
+import latticex.rosetta as rtt
+import csv
+import numpy as np
+
+rtt.set_backend_loglevel(1)
+np.set_printoptions(suppress=True)
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '2'
+np.random.seed(0)
+#激活协议
+rtt.activate("SecureNN")
+mpc_player_id = rtt.py_protocol_handler.get_party_id()
+```
+
+- 加载数据集
+
+这一部分数据集的加载与之前例子中的略有不同，之前加载时在`PrivateDataset`中使用了`load_data`方法，可同时加载特征数据与标签数据。但是，这种同时加载的方式仅支持一维的标签数据，`mnist`中的标签数据是使用`one-hot`编码的共有10维，所以此处使用了加载特征数据的方式加载了标签数据。关于`load_data`、`load_X`、`load_Y`的区别请查阅源代码。
+
+```python
+#加载数据集
+file_x = "./train_x.csv"
+file_y = "./train_y.csv"
+
+X_train = rtt.PrivateDataset(data_owner=(0, 1), label_owner=1).load_X(file_x, header=None)
```

Comment:
For such big dataset, using rtt.PrivateTextLineDataset to load data iteratively may be better.

Reply:
I tried the API `rtt.PrivateTextLineDataset`, but its return type is not `tensor`,when i call the function`_element_structure()`, the code throws `TypeError: 'TensorStructure' object is not callable`.

Reply:
Yes, this API can NOT used alone, you may check this [sample in tutorial](https://github.com/LatticeX-Foundation/Rosetta/blob/master/doc/TUTORIALS_CN.md#%E6%94%AF%E6%8C%81%E8%B6%85%E5%A4%A7%E6%95%B0%E6%8D%AE%E9%9B%86)

Reply:
I have read the [sample in tutorial](https://github.com/LatticeX-Foundation/Rosetta/blob/master/doc/TUTORIALS_CN.md#%E6%94%AF%E6%8C%81%E8%B6%85%E5%A4%A7%E6%95%B0%E6%8D%AE%E9%9B%86), but, this sample does not use loss function. In my sample, the loss function requests the parameter whose type is `narray`. However, 
 this API only return `tensor`. Alsoly, this `tensor` can not be converted into `narray`

Reply:
Yeah，I have updated this sample code with TF Optimizer in this [commit](https://github.com/LatticeX-Foundation/Rosetta/commit/17147c41bfd750e7bf67dafc15d787869508870a). You may refer to it now.

Reply:
I have changed the method of loading data.